pkgs/development/python-modules/unify/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchFromGitHub, untokenize }:
```

Expand this to one input per line, this makes diffing and merging easier.

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, untokenize
}:
```